webapp/channels/src/components/sidebar/sidebar_category/sidebar_category_menu/sidebar_category_menu.tsx

```diff
@@ -215,13 +212,14 @@ const SidebarCategoryMenu = (props: Props) => {
         props.openModal({
             modalId: ModalIdentifiers.EDIT_CATEGORY,
             dialogType: EditCategoryModal,
+            dialogProps: {},
```

Comment:
🗒️ 
The dialogProps is un-undefined param for EditCatgoryModal, and console was complaining about missing props.